lib/plugins/mongoose-encryption.js

```diff
@@ -360,11 +360,10 @@ var mongooseEncryption = function(schema, options) {
         cb();
     };
 
-    schema.methods.decryptSync = function() {
```

Comment:
Initially, `decryptSync` has no parameters, and were run in execution context of the object to be decrypted. Usually it's `Document` itself, but in `init` hook we need to decrypt `data` object.

We'll need access to Document later for `cf` support.

To keep backward compatibility, `decryptData` method extracted from `decryptSync`, which allows to decrypt object provided in first parameter, running in execution context of `Document`.

Reply:
Basically `decryptData` is same as `decryptSync`, but `this` changed to `data` parameter.